policybot/Makefile

```diff
@@ -5,7 +5,7 @@ HUB ?= gcr.io/istio-testing/policybot
 IMG := $(HUB):$(TAG)
 export GO111MODULE=on
 
-dockerrun := docker run -t -i --sig-proxy=true --rm -v $(shell pwd):/site -w /site "gcr.io/istio-testing/website-tools:2019-07-25"
+dockerrun := docker run -t -i --sig-proxy=true --rm -v $(shell pwd):/site -w /site "gcr.io/istio-testing/build-tools:2019-08-16T20-50-36"
```

Comment:
this part looks a little suspect, or atleast may need further attention.  Anyway approving for now, assuming the checks pas this should be reworked or removed entirely.

Reply:
I was originally envisioning several different bots, thus the 'bots' repo. In hindsight, I ended up with a single bot. I'm thinking of hoisting everything in the policybot/* directory one level above in the repo. Then we wouldn't have this local makefile and would just build from the top.